docs/guides/advise/_index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Can you please reword the commit message to precise this is related to the website?
When I first saw your PR I was like: "Hum... Isn't it already done?"

Reply:
I reworded it. Thanks!

Reply:
One small nit: `s/caterogirization/categorization/`.